src/app/collection/settings/tab/rules/collection-rules.component.ts

```diff
@@ -135,4 +144,21 @@ export class CollectionRulesComponent implements OnInit, OnDestroy {
   public trackByRuleName(index: number, rule: Rule): string {
     return rule.name;
   }
+
+  public openServiceOrder() {
+    this.store$
+      .pipe(
+        select(selectOrganizationByWorkspace),
+        map(organization => organization.code),
+        first()
+      )
+      .subscribe(code => {
+        this.store$.dispatch(
+          new RouterAction.Go({
+            path: ['/organization', code, 'detail'],
+            extras: {fragment: 'orderService'},
```

Comment:
What is fragment?

Reply:
Selected tab page, rendered as `#orderService` in url.

Reply:
Oo I See!